ospf6d/ospf6_top.h

```diff
@@ -158,6 +160,16 @@ struct ospf6 {
 	struct list *oi_write_q;
 
 	uint32_t redist_count;
+
+	/* Action for aggregation of external LSAs */
+	int aggr_action;
+
+#define OSPF6_EXTL_AGGR_DEFAULT_DELAY 5
+	/* For ASBR summary delay timer */
+	int aggr_delay_interval;
```

Comment:
Is the delay really necessary? Why do we have this knob?

If we are using timers/event it will already work OK to batch multiple configuration batching.

Reply:
Since LS_MIN_INTERVAL is 5 seconds as per RFC, we need to originate the LSAs at an interval of atleast 5 seconds. Also as you said it will be helpful in batching multiple config too. The user does not need to use this command if it does not want it but a method is provided in case user wants to modify it. If you see Cisco has the command "timer lsa arrival" which won't accept LSAs if arrived within this time interval. This knob will be useful in such cases as well.